gst/nnstreamer/include/nnstreamer_plugin_api_if.h

```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * GStreamer/NNStreamer Tensor-IF
```

Comment:
For internal API headers, you need to provide more descriptions: how to use this header.

Reply:
Description is added.  Thanks.